lib/rack/utf8_sanitizer.rb

```diff
@@ -61,17 +68,17 @@ def sanitize_rack_input(env)
       # https://github.com/rack/rack/blob/master/lib/rack/request.rb#L42
       # Logic borrowed from Rack::Request#media_type,#media_type_params,#content_charset
       # Ignoring charset in content type.
-      content_type   = env['CONTENT_TYPE']
-      content_type &&= content_type.split(/\s*[;,]\s*/, 2).first
+      content_type   = env[CONTENT_TYPE]
+      content_type &&= content_type.split(CONTENT_TYPE_SPLIT, 2).first
```

Comment:
That may be true. I wasn't as sure about adding the regexp constants and I'd be fine pulling them out or trying a profiler run with and without.